src/components/my-history/custom-my-history.vue

```diff
@@ -20,8 +20,24 @@
           >
             <HistoryIcon class="max-desktop:x-icon-lg" />
 
-            <div class="x-flex x-flex-col x-gap-2">
-              <p>{{ suggestion.query }}</p>
+            <div class="x-group x-flex x-flex-col x-gap-2">
+              <p class="group-hover:x-underline">{{ suggestion.query }}</p>
+              <div
```

Comment:
This is a pretty big chunk that is duplicated here and in the empathize, moving it to it's own component might be a good idea.